hack/all-in-one/microshift-aio-run

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+setsebool -P container_manage_cgroup true
+
+if ! /usr/bin/podman volume exists microshift-vol
+then
+  /usr/bin/podman volume create microshift-vol
+fi
+
+[[ -d /etc/microshift-aio ]] || mkdir /etc/microshift-aio
```

Comment:
what is this for?

Reply:
I wanted a file with the location of the KUBECONFIG like so:
```
cat <<EOF > /etc/microshift-aio/microshift-aio.conf
export KUBECONFIG=$(/usr/bin/podman volume inspect microshift-vol --format "{{.Mountpoint}}")/microshift/resources/kubeadmin/kubeconfig
EOF
```
I could not see a better way of getting that path from the service, but that is definitely extra and does not need to be included in the service. I figured it saves users from having to look up the path.

Reply:
I also could not figure out how to run `ExecStartPost=source /etc/microshift-aio/microshift-aio.conf` 
I tried `ExecStartPost=source`, `ExecStartPost=/bin/bash source` etc, nothing worked but there is def something that does, I gave up and pushed the commit but will keep looking

Reply:
@husky-parul if you find something that works or if we decide it's not necessary, you can remove it, feel free to squash/amend, too

Reply:
i see, that makes sense